tests/config/pytest_modules.yml

```diff
@@ -876,6 +876,7 @@ homer/annotatepeaks:
 
 homer/findpeaks:
   - modules/homer/findpeaks/**
+  - modules/homer/maketagdirectory/**
```

Comment:
Curious what this line is for

Reply:
The tests directly depend on maketagdirectory, and the module is going to consume maketagdirectory a good portion of the time. It's just catching any regressions introduced by changes to maketagdirectory.

Reply:
cool thanks for the info!